lib/backend/model/profile.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 var (
-	matchProfile = regexp.MustCompile("^/?calico/v1/policy/profile/([^/]+)/(tags|rules|labels)$")
+	matchProfile = regexp.MustCompile("^/?calico/v1/policy/profile/([^/]+)(/tags|/rules|/labels)?$")
```

Comment:
I'm not sure if this should be changed. It seemed reasonable to me especially with what the ProfileListOptions seems to expect.  My motivation for the change was purely for testing reasons as it allowed what I have with the Profile test.

Reply:
i don't think we used the actual matched values, rather we just used this to check a match, and the two variants of this regex both have positive matched on the same thing.

I assume you must be using the match values and you wanted "/tag" instead of "tag".

It does feel a little odd returning "/tag" rather than "tag" tbh.